test/net.js

```diff
@@ -327,11 +330,12 @@ tape('emits close', function (t) {
     server.close()
     t.ok(serverClosed)
     t.ok(clientClosed)
-    t.end()
   }
 })
 
-tape('flushes', function (t) {
+test('flushes', function (t) {
```

Comment:
This test currently causes a segfault:

```
PID 11349 received SIGSEGV for address: 0x0
0   segfault-handler.node               0x0000000112e5f09a _ZL16segfault_handleriP9__siginfoPv + 298
1   libsystem_platform.dylib            0x00007fff204ecd7d _sigtramp + 29
2   ???                                 0x0000000000000026 0x0 + 38
3   node                                0x000000010e6da35b _ZN4node6Buffer4DataEN2v85LocalINS1_5ValueEEE + 23
4   node                                0x000000010e6d2388 napi_get_buffer_info + 51
5   utp_native.node                     0x0000000112e3889a _ZL13on_utp_acceptP22utp_callback_arguments + 346
6   utp_native.node                     0x0000000112e40e7d _Z18utp_call_on_acceptP18struct_utp_contextP9UTPSocketPK8sockaddrj + 45
7   utp_native.node                     0x0000000112e3f33d utp_process_udp + 1405
8   utp_native.node                     0x0000000112e390b0 _ZL10on_uv_readP8uv_udp_slPK8uv_buf_tPK8sockaddrj + 144
9   libuv.1.dylib                       0x0000000110a1e375 uv__udp_io + 350
10  libuv.1.dylib                       0x0000000110a21773 uv__io_poll + 1861
11  libuv.1.dylib                       0x0000000110a11e46 uv_run + 359
12  node                                0x000000010e6470d7 _ZN4node13SpinEventLoopEPNS_11EnvironmentE + 255
13  node                                0x000000010e7333ef _ZN4node16NodeMainInstance3RunEPiPNS_11EnvironmentE + 97
14  node                                0x000000010e733082 _ZN4node16NodeMainInstance3RunEPKNS_16EnvSerializeInfoE + 126
15  node                                0x000000010e6d0039 _ZN4node5StartEiPPc + 216
16  libdyld.dylib                       0x00007fff204c2f3d start + 1
17  ???                                 0x0000000000000002 0x0 + 2
zsh: segmentation fault  node test/net.js
```

Reply:
@mafintosh This test now fails as only the first write makes it through. Is this the intended behaviour of streamx?